src/sssom/cli.py

```diff
@@ -30,7 +30,7 @@
     DEFAULT_VALIDATION_TYPES,
     PREFIX_MAP_MODES,
     SchemaValidationType,
-    SSSOMSchemaView,
+    _get_sssom_schema_object,
```

Comment:
Private means nobody should use this function outside of the package itself. Otherwise how could you use the same private function in multiple modules?

Reply:
Well in java you cant use it outside the class, so its much stricter. But I guess it makes some sense for python to widen this to package level. A Java programmer, which I am not, would tell you now that "the need of important a private method into multiple modules is a sign of a bad software design"; which I only very partially agree with